packages/camera/camera_windows/lib/camera_windows.dart

```diff
@@ -0,0 +1,424 @@
+
+    throw UnsupportedError(
+        'setFocusPoint() is not supported due to Win32 API limitations.');
+  }
+
+  @override
+  Future<double> getMinZoomLevel(int cameraId) async {
+    // TODO(jokerttu): Implement zoom level support.
+    // Value is returned to support existing implementations.
+    return 1.0;
+  }
+
+  @override
+  Future<double> getMaxZoomLevel(int cameraId) async {
+    // TODO(jokerttu): Implement zoom level support.
+    // Value is returned to support existing implementations.
+    return 1.0;
+  }
+
+  @override
+  Future<void> setZoomLevel(int cameraId, double zoom) async {
+    // TODO(jokerttu): Implement zoom level support.
+    throw UnimplementedError('setZoomLevel() is not implemented.');
+  }
+
+  @override
+  Future<void> pausePreview(int cameraId) async {
+    await _pluginChannel.invokeMethod<double>(
+      'pausePreview',
+      <String, dynamic>{'cameraId': cameraId},
+    );
+  }
+
+  @override
+  Future<void> resumePreview(int cameraId) async {
+    await _pluginChannel.invokeMethod<double>(
+      'resumePreview',
+      <String, dynamic>{'cameraId': cameraId},
+    );
+  }
+
+  @override
+  Widget buildPreview(int cameraId) {
+    return Texture(textureId: cameraId);
+  }
+
+  /// Returns the resolution preset as a nullable String.
+  String? _serializeResolutionPreset(ResolutionPreset? resolutionPreset) {
+    switch (resolutionPreset) {
+      case null:
+        return null;
+      case ResolutionPreset.max:
+        return 'max';
+      case ResolutionPreset.ultraHigh:
+        return 'ultraHigh';
+      case ResolutionPreset.veryHigh:
+        return 'veryHigh';
+      case ResolutionPreset.high:
+        return 'high';
+      case ResolutionPreset.medium:
+        return 'medium';
+      case ResolutionPreset.low:
+        return 'low';
+    }
+  }
+
+  /// Converts messages received from the native platform into camera events.
+  Future<dynamic> _handleCameraMethodCall(MethodCall call, int cameraId) async {
+    switch (call.method) {
+      case 'camera_closing':
+        _cameraEventStreamController.add(
+          CameraClosingEvent(
+            cameraId,
+          ),
+        );
+        break;
+      case 'video_recorded':
+        // This is called if maxVideoDuration was given on record start.
+        _cameraEventStreamController.add(
+          VideoRecordedEvent(
+            cameraId,
+            XFile(call.arguments['path'] as String),
+            call.arguments['maxVideoDuration'] != null
+                ? Duration(
+                    milliseconds: call.arguments['maxVideoDuration'] as int,
+                  )
+                : null,
+          ),
+        );
+        break;
+      case 'error':
+        _cameraEventStreamController.add(
+          CameraErrorEvent(
+            cameraId,
+            call.arguments['description'] as String,
+          ),
+        );
+        break;
+      default:
+        throw MissingPluginException();
```

Comment:
Why MissingPlugin rather than Unimplemented?

Reply:
No real reason; following same patters used on Android and iOS platforms at [here](https://github.com/flutter/plugins/blob/main/packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart#L534)
and also [here](https://github.com/flutter/plugins/blob/main/packages/webview_flutter/webview_flutter_platform_interface/lib/src/method_channel/webview_method_channel.dart#L79)

I can change it to `UnimplementedError `if that suits better.

Reply:
Strange, I'm not sure why it does that. UnimplementedError would be much less confusing (the plugin definitely isn't missing, and a missing plugin error message can easily lead people down the wrong path), so we should use it here.